contracts/interfaces/IPawnshop.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IPawnshop {
+    // Emitteed when a borrower calls borrow
```

Comment:
We can use natspec rather than just comment. What do you think?

Reply:
Yeah! Happy to natspec the comments. Would be good to get feedback from @Constantino on the interface first before spending time prettying